src/modules.c

```diff
@@ -2,17 +2,34 @@
 #include "modules.h"
 
 // Return value of temperature sensor in degrees C
-void read_temperature()
+float read_temperature()
 {
 
 }
 
+void tempSensor()
+{
+	
+}
+
 // Returns value of light sensor between 0 and 100
 float read_lights()
 {
-	ADMUX = _BV(MUX0);
-	ADCSRA = _BV(ADSC);
+	ADMUX |= _BV(MUX0);
+	ADCSRA |= _BV(ADSC);
 	loop_until_bit_is_clear(ADCSRA, ADSC);
-
-	return (float) ADCW / 1024 * 100;
+	
+	return (float)ADCW / 1024 * 100;
 }
+
+void lightSensor()
+{
+	if (read_lights() > 20.0)
```

Comment:
Can be replaced with something like this:
``PORTD = read_lights() > 20.0 ? 0xFF : 0x00;``